package.json

```diff
@@ -45,36 +39,36 @@
     "LICENSE"
   ],
   "dependencies": {
-    "@types/quill": "1.3.10",
-    "@types/react": "*",
-    "create-react-class": "^15.6.0",
     "lodash": "^4.17.4",
-    "prop-types": "^15.5.10",
-    "quill": "^1.2.6",
-    "react-dom-factories": "^1.0.0"
+    "quill": "^1.3.7",
+    "react": "^16.11.0",
+    "react-dom": "^16.11.0"
   },
   "peerDependencies": {
-    "react": "^0.14.9 || ^15.3.0 || ^16.0.0"
+    "react": "^16.0.0"
   },
   "devDependencies": {
+    "@types/lodash": "^4.14.146",
+    "@types/quill": "^1.3.7",
+    "@types/react": "^16.9.11",
+    "@types/react-dom": "^16.9.4",
     "blanket": "^1.2.3",
-    "chai": "^4.1.0",
-    "chai-enzyme": "^0.8.0",
-    "cheerio": "^1.0.0-rc.1",
-    "enzyme": "^2.9.1",
+    "chai": "^4.2.0",
+    "chai-enzyme": "^1.0.0-beta.1",
+    "cheerio": "^1.0.0-rc.3",
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.15.1",
     "jsdom": "^11.0.0",
```

Comment:
Upgrading this to ~15 might unblock some of the TODOs in the test file

Reply:
I tried jsdom 15.0.0, however I recall it had some compatibility issues with contenteditables. I didn't investigate further, just figure out that 11.x.x is the last version that doesn't break.

I am thinking this should be left as a follow-up ticket.